### DISCUSSION TITLE
Newbie query: Can Stellarium be used with VR tech (we have something or other at uni)

### DISCUSSION THREAD

**Discussion by PsychnStatsTutor** [Stellarium/stellarium#1864] [Q&A]
Sorry to be so vague~ I have no idea what VR software/hardware we have. I'm wondering if it's enough to use the desktop version

**Comment by gzotti** [comment]
We even less know what hardware you have. There is a plugin for Oculus Rift which interested parties with the right version can extend upon. Almost all other users of desktop systems can use the ordinary version.

**Comment by PsychnStatsTutor** [comment]
Thanks @gzotti~ I've emailed the campus IT lead, with a Stellarium link.
